packages/eslint-plugin-wdio/docs/rules/await-deep-selector.md

```diff
@@ -0,0 +1,27 @@
+# Missing await before a deep selector (wdio/await-deep-selector)
+
+`$('>>>selector')` calls must be prefixed with an `await`.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```js
+describe('my feature', () => {
+    it('should do something', async () => {
+        const myButton = $('>>>button');
+        expect(myButton).toBeDisplayed();
+    });
+});
+```
+
+Examples of **correct** code for this rule:
+
+```js
+describe('my feature', () => {
+    it('should do something', async () => {
+        const myButton = await $('>>>button');
+        expect(myButton).toBeDisplayed();
```

Comment:
Can we add an example for this alternative to:

```ts
const myButton = $('>>>button');
await expect(myButton).toBeDisplayed();
```

Reply:
This alternative doesn't work . We need to add await  here: const myButton = await $('>>>button');
if not we notice an error in the case where the element doesn't exist : 
[0-0] 2024-01-30T10:35:14.660Z ERROR webdriver: Request failed with status 400 due to invalid argument: invalid argument: invalid locator